app/controllers/restaurant_employees_controller.rb

```diff
@@ -2,4 +2,20 @@ class RestaurantEmployeesController < ApplicationController
   def index
     @restaurant = Restaurant.find(params[:restaurant_id])
   end
+
+  def new
+    @restaurant = Restaurant.find(params[:restaurant_id])
```

Comment:
I'm glad yours works in the parent_child controller. I think I moved mine back and forth between this and the orders controller Friday night/Saturday morning before deciding it should live here. I don't feel like I was too far off now.

I realize I didn't have anything in this method on mine, so I'm implementing it as well.